src/lib/openapi/spec/resource-limits-schema.ts

```diff
@@ -103,6 +104,13 @@ export const resourceLimitsSchema = {
             example: 300,
             description: 'The maximum number of segments allowed.',
         },
+        flags: {
+            type: 'integer',
+            minimum: 1,
+            example: 5000,
```

Comment:
I'd match the example with the actual limit since this is what most people will get anyway